concepticondata/conceptlists/Snodgrass-1980-260.tsv

```diff
@@ -0,0 +1,261 @@
+Snodgrass-1980-260-35	35	1465	BOX	Box
+Snodgrass-1980-260-36	36	1368	BREAD	Bread
+Snodgrass-1980-260-37	37	245	BROOM	Broom
+Snodgrass-1980-260-38	38	1821	BRUSH	Brush
+Snodgrass-1980-260-39	39	1070	BUS	Bus
+Snodgrass-1980-260-40	40	1791	BUTTERFLY	Butterfly
+Snodgrass-1980-260-41	41	757	BUTTON	Button
+Snodgrass-1980-260-42	42	2408	CAKE	Cake
+Snodgrass-1980-260-43	43	1871	CAMEL	Camel
+Snodgrass-1980-260-44	44	1273	CANDLE	Candle
+Snodgrass-1980-260-45	45	3202	CANNON	Cannon
+Snodgrass-1980-260-46	46	1288	CAP	Cap
+Snodgrass-1980-260-47	47	1111	CAR	Car
+Snodgrass-1980-260-48	48	3085	CARROT	Carrot
+Snodgrass-1980-260-49	49	1208	CAT	Cat
+Snodgrass-1980-260-50	50	2247	CATERPILLAR	Caterpillar
+Snodgrass-1980-260-51	51	3727	CELERY	Celery
+Snodgrass-1980-260-52	52	1627	CHAIN	Chain
+Snodgrass-1980-260-53	53	1476	CHAIR	Chair
+Snodgrass-1980-260-54	54	2765	CHERRY	Cherry
+Snodgrass-1980-260-55	55	1318	CHICKEN	Chicken
+Snodgrass-1980-260-56	56	1979	CHISEL	Chisel
+Snodgrass-1980-260-57	57	1349	CHURCH	Church
+Snodgrass-1980-260-58	58			Cigar
+Snodgrass-1980-260-59	59	969	CIGARETTE	Cigarette
+Snodgrass-1980-260-60	60	1323	CLOCK	Clock
+Snodgrass-1980-260-61	61			Clothespin
+Snodgrass-1980-260-62	62	1489	CLOUD	Cloud
+Snodgrass-1980-260-63	63			Clown
+Snodgrass-1980-260-64	64	603	COAT	Coat
+Snodgrass-1980-260-65	65	1475	COMB	Comb
+Snodgrass-1980-260-66	66	506	MAIZE	Corn
+Snodgrass-1980-260-67	67	3099	SOFA	Couch
+Snodgrass-1980-260-68	68	1007	COW	Cow
+Snodgrass-1980-260-69	69	1943	CROWN (OF HEAD)	Crown
+Snodgrass-1980-260-70	70	476	CUP	Cup
+Snodgrass-1980-260-71	71	1936	DEER	Deer
+Snodgrass-1980-260-72	72	1775	TABLE	Desk
+Snodgrass-1980-260-73	73	2009	DOG	Dog
+Snodgrass-1980-260-74	74	1659	DOLL	Doll
+Snodgrass-1980-260-75	75	1862	DONKEY	Donkey
+Snodgrass-1980-260-76	76	1567	DOOR	Door
+Snodgrass-1980-260-77	77			Doorknob
+Snodgrass-1980-260-78	78	474	DRESS	Dress
+Snodgrass-1980-260-79	79			Dresser
+Snodgrass-1980-260-80	80	908	DRUM	Drum
+Snodgrass-1980-260-81	81	1360	DUCK	Duck
+Snodgrass-1980-260-82	82	1905	EAGLE	Eagle
+Snodgrass-1980-260-83	83	1247	EAR	Ear
+Snodgrass-1980-260-84	84	1290	ELEPHANT	Elephant
+Snodgrass-1980-260-85	85	3088	ENVELOPE	Envelope
+Snodgrass-1980-260-86	86	1248	EYE	Eye
+Snodgrass-1980-260-87	87	1690	FENCE	Fence
+Snodgrass-1980-260-88	88	1303	FINGER	Finger
+Snodgrass-1980-260-89	89	227	FISH	Fish
+Snodgrass-1980-260-90	90	2623	FLAG	Flag
+Snodgrass-1980-260-91	91	239	FLOWER	Flower
+Snodgrass-1980-260-92	92	777	FLUTE	Flute
+Snodgrass-1980-260-93	93	1504	FLY (INSECT)	Fly
+Snodgrass-1980-260-94	94	1301	FOOT	Foot
+Snodgrass-1980-260-95	95	1845	BALL	Football
+Snodgrass-1980-260-96	96			Football helmet
+Snodgrass-1980-260-97	97	1302	FORK	Fork
+Snodgrass-1980-260-98	98	1312	FOX	Fox
+Snodgrass-1980-260-99	99	939	HORN (INSTRUMENT)	French horn
+Snodgrass-1980-260-100	100	503	FROG	Frog
+Snodgrass-1980-260-101	101	31	PAN	Frying pan
+Snodgrass-1980-260-102	102			Garbage can
+Snodgrass-1980-260-103	103	3089	GIRAFFE	Giraffe
+Snodgrass-1980-260-104	104	476	CUP	Glass 
```

Comment:
I would unmap "Glass" here, even if I support the idea. The problem is that we have in the European context a distinction between Tasse and Glas as containers, and I consider it useful to maybe add GLASS (CONTAINER FOR LIQUID) later as a concept to Concepticon.

Reply:
Sounds good, I will map it as a new concept and add it to Concepticon! Definition as per Cambridge? "a small container for drinks made of glass or similar material with a flat base and usually with no handle"? @LinguList